ts/src/flexible-event/main.ts

```diff
@@ -38,6 +41,16 @@ const options = parse<Arguments>({
     type: Number,
     defaultValue: 20,
   },
+  attribution_scope_limit: {
+    alias: 'a',
+    type: Number,
+    defaultValue: 0,
```

Comment:
I'm curious about this being defaulted to 0, whereas during parsing it defaults to null. Is there a reason we have to use null during parsing but can use 0 here? If so, is it a problem that the user can't specify `null` here? If not, can we change parsing to use 0 as well?

Reply:
I wasn't aware user can specify `null` here, could you elaborate on how to do that? 

yes, 0 is almost equivalent to `null`. the only difference is that we now will accept `attribution_scope_limit` as valid input, which will require quite some changes everywhere. I am ok to keep it as is.

Reply:
My point is that the user can't specify `null` here.

Reply:
I believe if user don't provide the value here, it's set to `undefined` in type script, right? `undefined` should be equivalent to null imo.

Reply:
What is the purpose of the `defaultValue` field if the value defaults to `undefined`?

Reply:
marked this arg as optional.